tasmota/include/tasmota_configurations_ESP32.h

```diff
@@ -204,19 +204,31 @@
 #undef USE_I2C
 #undef USE_HOME_ASSISTANT
 #define USE_TASMOTA_DISCOVERY                    // Enable Tasmota Discovery support (+2k code)
-#undef USE_COUNTER
+#undef USE_DOMOTICZ
+#undef USE_LIGHT
```

Comment:
Does removing light make the PWM channel used for the AIThinker LED unusable?

Reply:
Oops, yes